internal/test_utils/temp_dir_fixture.go

```diff
@@ -0,0 +1,62 @@
+package test_utils
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/windmilleng/wat/os/temp"
```

Comment:
wat

i think you meant the one in wmclient

Reply:
lol good catch